src/views/compute/DeployVM.vue

```diff
@@ -774,6 +780,9 @@ export default {
     },
     networkName () {
       return this.$route.query.name || null
+    },
+    showSecurityGroupSection () {
+      return this.networks.length > 0 && this.zone.securitygroupsenabled && this.hypervisor !== 'KVM'
```

Comment:
@utchoang Why is KVM excluded for SG Selection ?

Reply:
@davidjumani 
I refer to https://github.com/apache/cloudstack/blob/master/ui/scripts/instanceWizard.js#L177 then I think KVM does not support it.

Reply:
Got it! But in https://github.com/apache/cloudstack/blob/master/ui/scripts/ui-custom/instanceWizard.js#L1570 it does support it along with multiple IPs

Reply:
@davidjumani I understood. I'll remove `this.hypervisor !== 'KVM'` condition. Thanks.